index.md

```diff
@@ -48,6 +48,361 @@ def is_even(num):
     return num % 2 == 0
 ```
 
+### E1101: no-member {#E1101}
+
+We expect you to access an object(variable, function...) for an existent member.
+For example, you should not call a variable or function which is not defined.
+In the following example, the Rectangle class does not have the area()
+method(member).
+
+
+```python
+class Rectangle:
+    def __init__(self, width, height):
+        """
+        Initialize the Rectangle.
+
+        @type self: Rectangle
+        @type width: int
+        @type height: int
+        @rtype: None
+        """
+        self.width = width
+        self.height = height
+
+r = Rectangle(5, 4)
+
+# no such member in Rectangle class
+print(r.area())  # Error on this line
+```
+
+### E1102: not-callable {#E1102}
+
+We expect you to call an object only if it has been inferred to a callable object.
+In the following example, x is a variable with value 10. We should not call x()
+ because it is not callable.
+
+```python
+x = 10
+print(x())  # Error on this line
+```
+
+### E1111: assignment-from-no-return {#E1111}
+
+In Python, we expect you to assign to a function call with the infered function
+does return an object. This error is similar to E1128. In the following example,
+f() does not return anything. As a result, x has no value if we assign x f().
+Remember! Whenever you define a function, return some objects otherwise it
```

Comment:
It is perfectly acceptable to define a function which does not have a return value. Don't advise against doing so.